data/sql/headers.go

```diff
@@ -323,13 +333,25 @@ func (h *HeadersDb) GetAncestorOnHeight(hash string, height int32) (*dto.DbBlock
 	return bh[0], nil
 }
 
-// GetAllTips returns all tips from db.
+// GetAllTips returns headers whose hash does not appear as previous block hashes in any header within the highest 100 headers.
```

Comment:
@sirdeggen what actually is a business requirement for that method 
What we want to actually return from endpoint in an imaginary ideal world?
Which of the following should be included in the ideal solution:
1. tip of longest chain
2. tip of newest forked chain
3. tip of all forked chains
4. tip of all orphan chains